app/scss/pages/_rooms.scss

```diff
@@ -0,0 +1,149 @@
+.room {
+	&__cards {
+		padding: 50px 0;
+		&-title {
+			text-align: center;
+			font-size: 50px;
+    	font-family: AGaramondPro,sans-serif;
+    	@media (max-width : 720px) {
+				font-size: 40px;
+			}
+		}
+		&-text {
+			text-align: center;
+			@media (max-width : 550px) {
+				font-size: 15px;
+			}
+		}
+	}
+	&__card {
+		border: 2px solid #14274A;
+		margin-top: 50px;
+		.card {
```

Comment:
move to new line

**always make sure there is no nesting!!!**

**if you have nesting something you do incorrect - or class naming or scss styles!!!**

Reply:
done